pom.xml

```diff
@@ -8,11 +8,11 @@
         <version>2.9</version>
     </parent>
 
-    <artifactId>run-selector-api</artifactId>
+    <artifactId>run-selector</artifactId>
     <packaging>hpi</packaging>
-    <name>Run Selector API Plugin</name>
-    <version>0.1.0-SNAPSHOT</version>
-    <url>http://wiki.jenkins-ci.org/display/JENKINS/Run+Selector+API+Plugin</url>
+    <name>Run Selector Plugin</name>
```

Comment:
As next step, I'm planning to write a new pipeline step, that would look something like:
`def runWrapper = runSelector jobName: 'upstream-name', selector: [$class: 'selectorType', ...]`
This step would return a `RunWrapper` object to be used by other steps, for Pipeline usage.

In this case, it may be considered a standalone plugin, so I think that we should drop the *API* syntax from the name. Opinions?